x/interchainstaking/ibc_module.go

```diff
@@ -266,7 +266,7 @@ func (im IBCModule) OnRecvPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) ibcexported.Acknowledgement {
-	return channeltypes.NewErrorAcknowledgement("cannot receive packet via interchain accounts authentication module")
+	return channeltypes.NewErrorAcknowledgement(types.Errors{})
```

Comment:
https://github.com/cosmos/ibc-go/blob/48a6ae512b4ea42c29fdf6c6f5363f50645591a2/modules/core/04-channel/types/acknowledgement_test.go#L47

Reply:
fmt.Errorf("cannot receive packet via interchain accounts authentication module")

Reply:
Fix in https://github.com/notional-labs/quicksilver/commit/aab37f851670fde663ed1b04b299e6be033aed8d